src/db/table/user.sql

```diff
@@ -0,0 +1,15 @@
+-- User stores all the users that exist on the platform
+CREATE TABLE `user` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `username` varchar(45) NOT NULL,
+  `display_name` varchar(40) NOT NULL,
+  `email` varchar(70) NOT NULL,
+  `password` binary(60) NOT NULL,
```

Comment:
* Is there a reason we're using binary instead of char?
* Length should be 128 for SHA-512.